src/main/java/ch/astorm/jchess/util/UnicodePositionRenderer.java

```diff
@@ -0,0 +1,73 @@
+package ch.astorm.jchess.util;
+
+import ch.astorm.jchess.core.Color;
+import ch.astorm.jchess.core.Moveable;
+import ch.astorm.jchess.core.Position;
+import ch.astorm.jchess.core.entities.*;
+
+import java.io.PrintStream;
+
+public class UnicodePositionRenderer extends AbstractTextPositionRenderer {
+    private final  String lineSeparator = super.getLineSeparator();
```

Comment:
Still to be removed.

Reply:
Should I also remove [this line in the ASCII renderer](https://github.com/ctabin/jchess/pull/23/files#diff-0d5f8f5007ccaaee4554306c2d9d111a534ec362a415857ccbeec475fb0d0df6R123)?
I was trying to keep them consistent.

Reply:
OH nevermind, that was a local variable.